client/stylesheets/_toCat.scss

```diff
@@ -4446,6 +4448,7 @@ On click the images will be displayed at normal size to complete the effect
 .profileReceivedImg {
     width: 100% !important;
     height: 100%;
+    border-radius: 6px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces